lib/converters/v2.0.0/converter-v2-to-v1.js

```diff
@@ -368,10 +368,10 @@ _.assign(Builders.prototype, {
             name: item.name,
             collectionId: collectionId,
             method: item.request ? item.request.method : undefined,
-            currentHelper: currentHelper,
-            helperAttributes: helperAttributes
+            currentHelper: currentHelper
         };
 
+        helperAttributes && (request.helperAttributes = helperAttributes);
```

Comment:
`request.helperAttributes = helperAttributes` would set it to `undefined`, which will be ignored by the receiver right?

Reply:
Yes. In addition, the truthy-ness of `helperAttributes` is being checked first.